decidim-core/lib/decidim/core/engine.rb

```diff
@@ -32,6 +32,7 @@
 require "premailer/rails"
 require "nokogiri"
 require "geocoder"
+require "letter_opener_web"
```

Comment:
Yeah, we should only load this in development. The best way would probably be to load it in `decidim-dev`'s `railtie.rb`.